app/routes/administrative-units/administrative-unit/core-data.js

```diff
@@ -21,6 +21,7 @@ export default class AdministrativeUnitsAdministrativeUnitCoreDataRoute extends
         'is-associated-with',
         'resulted-from',
         'was-founded-by-organizations',
+        'kbo-administrative-unit',
```

Comment:
**Issue:** Add `kbo-administrative-unit` here is not needed, because `await administrativeUnit.kboAdministrativeUnit` in `app/routes/administrative-units/administrative-unit/core-data/index.js` will load it anyway. Moreover, this model is also used in the `/edit` route, where the `kbo-administrative-unit` is not require.